docs/index.md

```diff
@@ -1,10 +1,63 @@
-# ESSdiffraction
+:::{image} _static/logo.svg
+:class: only-light
+:alt: ESSdiffraction
+:width: 60%
+:align: center
+:::
+:::{image} _static/logo-dark.svg
+:class: only-dark
+:alt: ESSdiffraction
+:width: 60%
+:align: center
+:::
 
-<span style="font-size:1.2em;font-style:italic;color:#5a5a5a">
+```{raw} html
+   <style>
+    .transparent {display: none; visibility: hidden;}
+    .transparent + a.headerlink {display: none; visibility: hidden;}
+   </style>
+```
+
+```{role} transparent
+```
+
+# {transparent}`ESSdiffraction`
+
+<span style="font-size:1.2em;font-style:italic;color:var(--pst-color-text-muted)">
```

Comment:
I changed this to use a variable for the color so that it adapts to the dark theme.